streamer/src/packet.rs

```diff
@@ -30,16 +30,20 @@ pub fn recv_from(obj: &mut Packets, socket: &UdpSocket, max_wait_ms: u64) -> Res
         );
         match recv_mmsg(socket, &mut obj.packets[i..]) {
             Err(_) if i > 0 => {
+                break;
+                /*
                 if start.elapsed().as_millis() as u64 > max_wait_ms {
                     break;
                 }
+                */
```

Comment:
Is this safe to delete? Sorry, I'm lacking some context here

Reply:
This was just a quick test to see how this would show up in packet timing data. Added results to the description. I'll clean this up before moving the review out of draft status.